pkgs/development/interpreters/groovy/default.nix

```diff
@@ -15,8 +15,10 @@ stdenv.mkDerivation rec {
 
   installPhase = ''
     mkdir -p $out
+    mkdir -p $out/share/groovy
     rm bin/*.bat
-    mv * $out
+    mv {bin,conf,embeddable,grooid,indy,lib} $out
+    mv {licenses,LICENSE,NOTICE} $out/share/groovy
```

Comment:
I haven't seen any explicitly stated policy on the matter but I do see some references to `share/doc` in the [Nixpkgs manual](https://nixos.org/nixpkgs/manual/).